lib/notification_service/notification_service/cli/commands/event_command.py

```diff
@@ -0,0 +1,132 @@
+            return
+        return f(*args, **kwargs)
+
+    return cast(T, wrapper)
+
+
+@action_logging
+def list_events(args):
+    """List events at the command line"""
+    client = NotificationClient(server_uri=args.server_uri)
+    events = client.list_events(key=args.key,
+                                namespace=args.namespace,
+                                version=args.begin_version,
+                                event_type=args.event_type,
+                                start_time=args.begin_time,
+                                sender=args.sender)
+    NotificationConsole().print_as(
+        data=events,
+        output=args.output,
+        mapper=lambda event: {
+            "namespace": event.namespace,
+            "key": event.key,
+            "value": event.value,
+            "event_type": event.event_type,
+            "sender": event.sender,
+            "create_time": event.create_time,
+            "context": event.context,
+            "version": event.version,
+        },
+    )
+
+
+@action_logging
+def count_events(args):
+    """Count events at the command line"""
+    client = NotificationClient(server_uri=args.server_uri)
+    res = client.count_events(key=args.key,
+                              namespace=args.namespace,
+                              version=args.begin_version,
+                              event_type=args.event_type,
+                              start_time=args.begin_time,
+                              sender=args.sender)
+    print(res[0])
+
+
+@action_logging
+def listen_events(args):
+    """Listen events at the command line"""
+    class CliWatcher(EventWatcher):
+        def __init__(self, queue):
+            self.queue = queue
+
+        def process(self, events: List[BaseEvent]):
+            for e in events:
+                self.queue.put(e)
+
+    client = NotificationClient(server_uri=args.server_uri)
+    event_queue = Queue()
+    client.start_listen_event(key=args.key,
+                              namespace=args.namespace,
+                              version=args.begin_version,
+                              event_type=args.event_type,
+                              start_time=args.begin_time,
+                              sender=args.sender,
+                              watcher=CliWatcher(event_queue))
+    try:
+        while True:
+            print(event_queue.get())
+    except KeyboardInterrupt:
+        pass
+
+
+@action_logging
+def send_event(args):
+    """Send an event at the command line"""
+    client = NotificationClient(server_uri=args.server_uri,
+                                default_namespace=args.namespace,
+                                sender=args.sender)
+    if not args.value:
+        print("Arguments --value not set. See `notification event send {} -h`")
+    event = BaseEvent(
+        key=args.key,
+        value=args.value,
+        event_type=args.event_type,
+        context=args.context)
+    client.send_event(event)
```

Comment:
Adds the log for the completion for sending event.

Reply:
Added